### DISCUSSION TITLE
[BUG] Vuetify Issue Creator : impossible to click on repro template links

### DISCUSSION THREAD

**Discussion by ThomasKientz** [vuetifyjs/vuetify#13639] [General] [answered]
It is no possible to click on the reproduction template links :

<img width="784" alt="Capture d’écran 2021-05-12 à 11 06 32" src="https://user-images.githubusercontent.com/10157845/117949203-2961f180-b312-11eb-88d4-d3ecbc45835b.png">

**Comment by jacekkarczmarczyk** [comment, marked as answer]
Created an issue in https://github.com/vuetifyjs/vuetify-issue-helper

**Reply by jacekkarczmarczyk** [reply]
Issue creator has been updated
